### ISSUE TITLE
Adding sentences to training data document for LUIS

### ISSUE THREAD

**Issue by sammy1997** [systers/sysbot#111]
## Description
As a contributor,
I need to add sentences to the given topics in the doc( doc link under Mocks), which are similar in meaning to the ones already there on the doc
so that the bot can be trained on LUIS.

## Mocks
https://docs.google.com/document/d/1mgTq9etOVN95Dic6AUl-XrQZkoG0tKXQbt7tfIgDwAA/edit?usp=sharing

### Update [Required]
- [ ] Add more sentences to the given category
### Enhancement to Update [Optional]
- [ ] Add more topics that you think are frequently asked by newcomers and add sentences to that

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
varying hours

**Comment by sammy1997** [comment]
@sys-bot approve

**Comment by sammy1997** [comment]
@sara-02 @ramitsawhney27 Can you please approve this? This is as per we discussed on the channel

**Comment by GeVic** [comment]
@sammy1997 Does all category needed to be added? And how much (approx)  should be added?

**Comment by sammy1997** [comment]
What categories are you talking about @GeVic ? I don't get it. If you want to add a new category, then you should typically add 20-25 sentences to it. Also, pick the sentences as far as possible from Slack channels

**Comment by GeVic** [comment]
@sammy1997 I meant to ask, we have to add more sentences to the existing topics?

**Comment by sammy1997** [comment]
Yes. That's the basic aim of the issue. We aim to keep this one open for data collection. If not anything else, this would help later while using NLP to train the bot.

**Comment by GeVic** [comment]
@sammy1997  Yeah! That's a good idea.

**Comment by anshima1** [comment]
Hello. I have added 10+ new sentences in the given google document. I hope they are adequate for training the bot. Let me know if there are any corrections.
